cmd/preflight/cmd/check_container.go

```diff
@@ -257,6 +270,38 @@ func validateCertificationProjectID(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// checkForNewerReleaseVersion checks if there is a newer release available
+func checkForNewerReleaseVersion(cmd *cobra.Command) {
+	logger, err := logr.FromContext(cmd.Context())
```

Comment:
I think using `logr.FromContextOrDiscard` might be a better choice, that way we don't have to worry about errors. But I'll let other's chime in as well.